freqtrade/templates/FreqaiExampleStrategy.py

```diff
@@ -63,45 +63,43 @@ def populate_any_indicators(
         :param informative: the dataframe associated with the informative pair
         """
 
-        coin = pair.split('/')[0]
```

Comment:
We should do the same improvement to FreqaiExampleHybridStrategy (basically align it to the new behavior)

Reply:
Done.